util.js

```diff
@@ -192,23 +196,44 @@ function fetch(url, options) {
         else {
             if (!options.nolog) console.log('Fetch: ' + url);
         }
-        return baseFetch(url, options)
+
+        // To overcome transient network errors, we'll fetch the same URL again
+        // a few times before we surrender when a network error occurs, letting
+        // a few seconds elapse between attempts
+        function fetchWithRetry(url, options, remainingAttempts) {
+            return (remainingAttempts <= 0) ?
+                baseFetch(url, options) :
+                baseFetch(url, options).catch(err => {
+                    if (!options.nolog) console.warn('Fetch attempt failed: ' + url);
+                    return new Promise((resolve, reject) => {
+                        setTimeout(function () {
+                            fetchWithRetry(url, options, remainingAttempts - 1)
+                                .then(resolve)
+                                .catch(reject);
+                        }, 2000 + Math.floor(Math.random() * 8000));
```

Comment:
out of curiosity, why the random length for the timeout?

Reply:
I guess I cannot make up my mind as to how long the code should wait before retrying, because I don't know the cause of the "socket hang up" message. I thought it would make sense to introduce an arbitrary delay not to hit the server again at specific intervals. But now that you make me think about it, I don't quite see what that could bring.

Typically, I do not know where the "svgwg.org" Web site is hosted, but perhaps it's running on a virtual machine that goes to sleep after some time in the absence of requests and that first needs to restart when a new request comes in, which could make it timeout on the first request. A random length won't help there, we just need to retry after 5 seconds. I'll drop the random part.

Reply:
Ah, I see you already merged this with the random counter. Well, no big deal.